patches/api/0352-Add-system-prop-for-disabling-config-comment-parsing.patch

```diff
@@ -0,0 +1,49 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Thu, 30 Dec 2021 16:35:56 -0800
+Subject: [PATCH] Add system prop for disabling config comment parsing
```

Comment:
```suggestion
Subject: [PATCH] Add system prop for disabling config comment parsing by default

Allows for legacy plugins to continue to work without changes by setting `Paper.parseYamlCommentsByDefault` to false
```

Reply:
Done, didnt do exactly that commit msg cause I don't like when the patch file cuts off abruptly, but I think it gets the msg across